File: Backend/htbp_parent/service/service_cmn/src/main/java/com/seclib/htbp/cmn/service/DictService.java
package com.seclib.htbp.cmn.service;

import com.baomidou.mybatisplus.extension.service.IService;
import com.seclib.htbp.model.cmn.Dict;
import com.seclib.htbp.model.hosp.HospitalSet;
import org.springframework.web.multipart.MultipartFile;

import javax.servlet.http.HttpServletResponse;
import java.util.List;

public interface DictService extends IService<Dict> {
    List<Dict> findChildData(Long id);

    void exportDictData(HttpServletResponse response);

    void importDictData(MultipartFile file);

    String getDictName(String s, String value);

    List<Dict> findByDictCode(String dictCode);
}

File: Backend/htbp_parent/service/service_cmn/src/main/java/com/seclib/htbp/cmn/service/DictService.kt
package com.seclib.htbp.cmn.service

import com.baomidou.mybatisplus.extension.service.IService
import com.seclib.htbp.model.cmn.Dict
import javax.servlet.http.HttpServletResponse
import org.springframework.web.multipart.MultipartFile

interface DictService : IService<Dict?> {
    fun findChildData(id: Long?): List<Dict?>?
    fun exportDictData(response: HttpServletResponse?)
    fun importDictData(file: MultipartFile?)
    fun getDictName(s: String?, value: String?): String?
    fun findByDictCode(dictCode: String?): List<Dict?>?
}